tests/helpers/parsers.js

```diff
@@ -105,7 +105,7 @@ const parsers = {
           && {
             errors: testObject.errors.map(
               (errorObject) => {
-                const nextSuggestions = errorObject.suggestions && {
+                const nextSuggestions = errorObject.suggestions && typeof errorObject.suggestions !== 'number' && {
```

Comment:
This is to support https://github.com/jsx-eslint/eslint-plugin-react/pull/3759/commits/e555df7c143c30c1a297012a3304a8fdf3c0bcff#diff-8b29603f921c88a33d76f41f8915e96034ccb626ba59608ab8d3c874a9aa667bR504